package.json

```diff
@@ -19,7 +19,7 @@
     "validate:test:sequential": "npm run test --workspaces --if-present",
     "validate:lint-concurrent:nimble-components": "npm run lint-concurrent -w packages/nimble-components",
     "validate:test-concurrent:nimble-components": "npm run test-concurrent -w packages/nimble-components",
-    "storybook": "concurrently -n nimble,spright,storybook \"npm run build-components:watch -w @ni/nimble-components\" \"npm run build-components:watch -w @ni/spright-components\" \"npm run start -w @ni-private/storybook\""
+    "storybook": "npm run start -w @ni-private/storybook"
```

Comment:
@rajsite Just noticed this no longer builds/watches nimble and spright components. It won't work anymore?

Reply:
Great. Just confirmed. I think my nimble-components build was broken, and I made source fixes then ran `npm run storybook`. I expected it to build nimble-components, so I was surprised when it didn't.